src/ssl.c

```diff
@@ -6963,9 +6984,14 @@ long wolfSSL_CTX_set_session_cache_mode(WOLFSSL_CTX* ctx, long mode)
     if (mode == SSL_SESS_CACHE_OFF)
         ctx->sessionCacheOff = 1;
 
-    if (mode == SSL_SESS_CACHE_NO_AUTO_CLEAR)
+    if ((mode & SSL_SESS_CACHE_NO_AUTO_CLEAR) != 0)
```

Comment:
Nginx ORs the flags together:
/ngx_event_openssl.c:2226:
        SSL_CTX_set_session_cache_mode(ssl->ctx,
                                       SSL_SESS_CACHE_SERVER
                                       |SSL_SESS_CACHE_NO_AUTO_CLEAR
                                       |SSL_SESS_CACHE_NO_INTERNAL_STORE);

So it is expected that multiple bits can be set. (SSL_SESS_CACHE_NO_AUTO_CLEAR = 0x08)